web_server/users/templates/users/mandala.html

```diff
@@ -0,0 +1,352 @@
+<!--
+
+Copyright (c) 2020 - YIFEI BAI - https://codepen.io/soju22/pen/gEYJoO
+
+Permission is hereby granted, free of charge, to any person 
+obtaining a copy of this software and associated documentation 
+files (the "Software"), to deal in the Software without restriction,
+ including without limitation the rights to use, copy, modify, 
+merge, publish, distribute, sublicense, and/or sell copies of 
+the Software, and to permit persons to whom the Software is 
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall 
+be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, 
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES 
+OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND 
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT 
+HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, 
+WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, 
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
+DEALINGS IN THE SOFTWARE.
+
+-->
+
+
+
+
+
+{% load static %}
+<html lang="en">
+
+<head>
+      <meta charset="UTF-8">
+
+      <script src='https://cdn.jsdelivr.net/npm/vue'></script>
+      <script src='https://cdnjs.cloudflare.com/ajax/libs/d3/5.9.1/d3.min.js'></script>
+      <link rel="stylesheet" href="{% static '/users/css/mandala.css' %}">
```

Comment:
Okay so I'm also seeing the vue and d3 dependencies duplicated here. Are we using Vue and D3 in places other than this mandala script? If not we might be leaking dependencies to other pages which could slow load times for other pages unnecessarily.